packages/viewer/src/modules/map/components/cesium/CesiumMap.vue

```diff
@@ -38,7 +39,7 @@ import useUIStore from '@/store/modules/ui'
 
 const dispatcher: ActionDispatcher = { name: 'CesiumMap.vue' }
 
-let viewer: Viewer | undefined
+const viewer = ref<Viewer | undefined>(undefined)
```

Comment:
No, please no ref around that, that is a huge performance issue.
Mark it as raw if you want to wrap some sort of lightweight Vue reactivity, but do not `ref` big framework instances (such as OL or Cesium)

Or if really necessary to have a reactive object here (I think we can get by without if we place good `v-if` and `if (viewer)` guards) you could use a [shallowRef](https://vuejs.org/api/reactivity-advanced.html#shallowref) instead

The problem with a ref is that any part of the Cesium instance will be wrapped in a reactivity proxy (and Cesium has a huge object with tons of props, so that means thousands of proxy...)